data/net/minecraft/world/entity/EquipmentSlot.mapping

```diff
@@ -12,3 +12,5 @@ CLASS net/minecraft/world/entity/EquipmentSlot
 		ARG 1 slotIndex
 	METHOD getFilterFlag ()I
 		COMMENT Gets the actual slot index.
+	METHOD getIndex (I)I
+		ARG 1 increment
```

Comment:
Perhaps, instead of `increment`, it should be something like `baseIndex`? As I understand this method, it returns the inventory index of this equipment slot relative to the base inventory index.